amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseRenewerTest.java

```diff
@@ -55,7 +56,8 @@
     private LeaseRefresher leaseRefresher;
 
     private static Lease newLease(String leaseKey) {
```

Comment:
Is there any validation in the test cases that Lease contains the hash range available in the Shard information ?  CAn we add any assert statements for it